staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/status.go

```diff
@@ -46,6 +46,8 @@ func ErrorToAPIStatus(err error) *metav1.Status {
 				status.Code = http.StatusInternalServerError
 			}
 		}
+		status.Kind = "Status"
+		status.APIVersion = "v1"
```

Comment:
Why isn't our machinery doing this for us? Because we are constructing a versioned one directly?

Reply:
This method can be used outside the layer of the api server (where we don't have a codec), for errors which prevented us from even getting a code.  There's a fallback path that uses raw JSON serialization.

Reply:
Also, we'd have to hardcode a scheme and we don't quite have a generic one for meta.k8s.io yet.  I'm going to have add one at some point to deal with versioned meta types.